src/conn/conn_open.c

```diff
@@ -102,6 +101,8 @@ __wt_connection_close(WT_CONNECTION_IMPL *conn)
 
     /* The eviction server is shut down last. */
     WT_TRET(__wt_evict_destroy(session));
+    /* The eviction server must behind __wt_evict_destroy, see WT-11878 */
```

Comment:
The comment seems a bit unclear to me. Also, we don't generally reference tickets in comments unless they are FIXMEs.  I'd suggest:
```
/* The capacity server can only be shut down after all I/O is complete. */
```
Anyone that needs more detail can look at `git blame` and get to PRs and tickets.